### PR TITLE
[RM CH02] Renumbering CH02 RM

### PR DESCRIPTION
**Pull request by ASawwaf**

### CODE REVIEW

**Review by TFredberg — CHANGES_REQUESTED** [review]
ToC changed, but not the Chapter headings

**Review by walterkozlowski — APPROVED** [review]

**Review by petorre — APPROVED** [review]

**Review by ulikleber — CHANGES_REQUESTED** [review]
Chapter titles also need to be changed.

File: doc/ref_model/chapters/chapter02.md [hunk 1]
```diff
@@ -3,8 +3,9 @@
 
 ## Table of Contents
 * [2.1 Workloads Collateral](#2.1)
-* [2.2 Analysis](#2.2)
-* [2.3 Cloud Infrastructure Profiles](#2.3)
+* [2.2 Use cases](#2.2)
+* [2.3 Analysis](#2.2)
+* [2.4 Cloud Infrastructure Profiles](#2.3)
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter02.md:8]:
```suggestion
* [2.4 Cloud Infrastructure Profiles](#2.4)
```

**Comment by wmk-admin** [comment]
@TFredberg, are you OK now with this PR and approving it?

**Comment by TFredberg** [comment]
The ToC text is OK, but the links points to the wrong sub chapters.
The Sub chapters are numbered 2.1, 2.2, 2.2 and 2.3 which is wrong.

**Comment by walterkozlowski** [comment]
I am closing this PR and will make the changes appropriately in a different PR, using the latest merged contents.

**Pull request closed without merging**